toolkit/imageconfigs/baremetal.json

```diff
@@ -58,7 +58,8 @@
                 "default": "kernel"
             },
             "AdditionalFiles": {
-                "additionalconfigs/macros.excludedocs": "/usr/lib/rpm/macros.d/macros.excludedocs"
+                "additionalconfigs/macros.excludedocs": "/usr/lib/rpm/macros.d/macros.excludedocs",
+                "additionalfiles/baremetal-marker": "/etc/os-source"
```

Comment:
It would probably be good to have the `baremetal-marker` filename match the `os-source` filename. For example, `os-source-baremetal`.